src/screens/Game/GamePage/index.tsx

```diff
@@ -509,12 +525,16 @@ export default function GamePage(): JSX.Element | null {
       }
 
       if (installPath === 'import' && gameInfo.is_game) {
-        const { filePaths } = await showOpenDialog({
+        const { filePaths, canceled } = await showOpenDialog({
```

Comment:
```suggestion
        const { filePaths, cancelled } = await showOpenDialog({
```

Reply:
That's weird because this came from the electron api. So it might be wrong on their side. Changing it to cancelled doesn't work.

Reply:
Yeah, check https://www.electronjs.org/docs/api/dialog#dialogshowopendialogsyncbrowserwindow-options

Reply:
Weird, they made a typo I guess.

Reply:
or no:
`canceled or cancelled is the past tense of the verb to cancel. Both spellings are correct; Americans favor canceled (one L), while cancelled (two Ls) is preferred in British English and other dialects. `

Reply:
Damn didn't know about this one, I generally always right US English while coding, and English (India) normally. Noted. 👍🏾